WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -9,16 +9,29 @@
 import android.widget.AdapterView;
 import android.widget.ListView;
 
+import com.android.volley.VolleyError;
+
 import org.wordpress.android.R;
+import org.wordpress.android.WordPress;
+import org.wordpress.android.models.Blog;
 import org.wordpress.android.models.Person;
 import org.wordpress.android.ui.ActivityLauncher;
+import org.wordpress.android.ui.accounts.BlogUtils;
+import org.wordpress.android.ui.people.utils.PeopleUtils;
 
 public class PeopleManagementActivity extends AppCompatActivity {
 
+    private int mLocalBlogId = BlogUtils.BLOG_ID_INVALID;
+    private Blog mBlog;
+    private PeopleAdapter mPeopleAdapter;
+
```

Comment:
I might be missing some insight in future PRs around the feature but, the current code below doesn't seem to benefit from mLocalBlogId and mBlog being global variables since those are basically used "once" in `onCreate`. Notice that `refreshUsersList` could accept those as parameters.

In contrast, there is merit in keeping mPeopleAdapter global as that one is used in an asynchronously (in a callback) as well.

If you choose to keep all as global, perhaps you could consider keeping the result of `mBlog.getDotComBlogId()` as global var as that's the only field needed from the mBlog object.

All in all, the main reason i'm raising this comment is because global variables are the usual culprits in device-rotation bugs so, keeping the global state as lean as possible is generally good. Cheers!